src/test/java/org/broadinstitute/hellbender/engine/dataflow/ReadsPreprocessingPipelineTestData.java

```diff
@@ -188,6 +189,25 @@ public static GATKRead makeRead(String contig, int start, int length, int i, Cla
         }
     }
 
+    public static List<GATKRead> makeReferenceShardBoundaryReads( final int numContigs, final int numShardsPerContig, final Class<?> readImplementation ) {
```

Comment:
Also had an off-by-one error in this method that I've now fixed (shardEnd was actually the start of the next shard).